numpy/core/src/multiarray/compiled_base.dispatch.c

```diff
@@ -0,0 +1,94 @@
+/**
+ * @targets $maxopt baseline
+ * SSE2 AVX2 AVX512F
+ * VSX VSX2
+ * NEON ASIMDDP
+ */
+#include "compiled_base.h"
+
+/*
+ * This function packs boolean values in the input array into the bits of a
+ * byte array. Truth values are determined as usual: 0 is false, everything
+ * else is true.
+ */
+NPY_NO_EXPORT void NPY_CPU_DISPATCH_CURFX(simd_compiled_base_pack_inner)
+(const char *inptr, npy_intp element_size,  npy_intp n_in, npy_intp in_stride, char *outptr, npy_intp n_out, npy_intp out_stride, char order)
+{
+    /*
+     * Loop through the elements of inptr.
+     * Determine whether or not it is nonzero.
+     *  Yes: set corresponding bit (and adjust build value)
+     *  No:  move on
+     * Every 8th value, set the value of build and increment the outptr
+     */
+    npy_intp index = 0;
+    int remain = n_in % 8;              /* uneven bits */
+
+#ifdef NPY_SIMD
```

Comment:
```suggestion
#if NPY_SIMD
```

Reply:
The CI is weild for the error `compiled_base.dispatch.avx512f.obj : error LNK2001: unresolved external symbol _cvtmask64_u64`, which should be able to discover.